microservices/services/dictionary/service/src/main/frontend/src/pages/DataDictionary.vue

```diff
@@ -264,8 +267,64 @@ onMounted(() => {
   .catch((reason) => {
     console.log('Error fetching and formatting rows: ' + reason);
   });
+
+  // This block is similar to watch() methods below, but handles the search when initially set not added.
```

Comment:
I'm not following this comment fully, I think this is what you meant?
```suggestion
  // This block is similar to watch() methods below, but handles the search when initially set via the URL.
```

Reply:
Yeah that works, thanks